config/configuration.yml

```diff
@@ -21,15 +21,10 @@ default:
   email_delivery:
     delivery_method: :smtp
     smtp_settings:
-      enable_starttls_auto: <%= ENV['SMTP_ENABLE_STARTTLS_AUTO'] %>
+      enable_starttls_auto: true
       address: <%= ENV['SMTP_ADDRESS'] %>
-      port: <%= ENV['SMTP_PORT'] %>
+      port: <%= ENV['SMTP_PORT'] || 587 %>
       domain: <%= ENV['SMTP_DOMAIN'] %>
-      authentication: <%= ENV['SMTP_AUTHENTICATION'] %>
-      user_name: <%= ENV['SMTP_USER_NAME'] %>
-      password: <%= ENV['SMTP_PASSWORD'] %>
- 
-  # ==== Sendmail command
-  #
-  #  email_delivery:
-  #    delivery_method: :sendmail
+      authentication: :plain
```

Comment:
If this value is always `plain`, `SMTP_AUTHENTICATION=plain # (or login)` line is not necessary in `.env.example`.